includes/apple-exporter/class-theme.php

```diff
@@ -1552,12 +1563,16 @@ public function get_last_error() {
 	}
 
 	/**
-	 * Get the computed layout columns.
+	 * If user override enabled return layout_columns theme value.
+	 * Else return the computed layout columns value.
 	 *
 	 * @access public
 	 * @return int The number of layout columns to use.
 	 */
 	public function get_layout_columns() {
+		if ( 'yes' === $this->get_value( 'layout_columns_override' ) ) {
+			return $this->get_value( 'layout_columns' );
```

Comment:
We should capture the layout columns value and check to see if it's non-zero here as well:
```suggestion
		if ( 'yes' === $this->get_value( 'layout_columns_override' ) && ! empty( $this->get_value( 'layout_columns' ) ) ) {
			return $this->get_value( 'layout_columns' );
```